sandbox/Popups.qml

```diff
@@ -0,0 +1,38 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+Column {
+    spacing: 20
+
+    StatusButton {
+        text: "StatusModal"
+        onClicked: modalExample.open()
+    }
+
+    StatusModal {
+            id: modalExample
+        anchors.centerIn: parent
+        visible: true
+        header.image.source: "https://pbs.twimg.com/profile_images/1369221718338895873/T_5fny6o_400x400.jpg"
+        header.title: "Header"
+        header.subTitle: "SubTitle"
+        header.detailsButtonSettings.name: "info"
+        showBack: true
```

Comment:
Do we know for sure that there can only be a "back button" in the left part of the footer? How about otherwise, we keep that open so consumers can decide what kind of button(s) there is?

Could be something like:

```
footer.leftButtons: [
   StatusButton {} 
]
footer.rightButtons: [
  StatusButton {}
]
```

On top of that, but just a nice to have, is to give consumers full control over what the footer even looks like by doing:

```
footer: ...
```
^ no big deal if we do that now.